napalm_eos/eos.py

```diff
@@ -204,18 +204,26 @@ def rollback(self):
         commands.append('write memory')
         self.device.run_commands(commands)
 
+    def _get_version(self):
```

Comment:
Isn't this method a bit hardcore? Is just one line :S

`result = self.device.run_commands(commands)[0]`


I personally think it hurts readability as I would expect a helper method would do more than this. And it also forces you to do several API calls instead of making one with several commands (particularly painful if you are operating devices with a high latency).